web/client-js/src/components/TheHeader.vue

```diff
@@ -35,7 +35,6 @@
 </template>
 
 <script lang="ts">
-
```

Comment:
ESLint asked to remove this line. Adding the `// eslint-disable-line` and explaining it seems unreasonable in this case.